apex/contrib/test/group_norm/test_group_norm.py

```diff
@@ -89,10 +89,10 @@ def verify_group_norm(self,
             dx_tst, dw_tst, db_tst = [t.grad.clone() for t in [x, weight, bias]]
 
         # compare
-        torch.testing.assert_close(y_tst, y_ref, atol=4e-2, rtol=0)
-        torch.testing.assert_close(dx_tst, dx_ref, atol=4e-2, rtol=0)
-        torch.testing.assert_close(dw_tst, dw_ref, atol=4e-2, rtol=0)
-        torch.testing.assert_close(db_tst, db_ref, atol=4e-2, rtol=0)
+        torch.testing.assert_close(y_tst,  y_ref,  atol=7e-2, rtol=0)
+        torch.testing.assert_close(dx_tst, dx_ref, atol=7e-2, rtol=0)
+        torch.testing.assert_close(dw_tst, dw_ref, atol=7e-2, rtol=0)
+        torch.testing.assert_close(db_tst, db_ref, atol=7e-2, rtol=0)
```

Comment:
is this for the large tensor?

Reply:
yes, float reduction accuracy error would grow up with larger tensor